src/modelio.cs

```diff
@@ -1068,6 +1103,16 @@ interface MDLObjectContainerComponent : MDLComponent, INSFastEnumeration
 		[Export ("removeObject:")]
 		void RemoveObject (MDLObject @object);
 
+		[iOS (10,3), TV (10,2), Mac (10,12,4)]
+		[Abstract]
```

Comment:
breaking change, you cannot add `abstract` methods as existing subclasses won't have them